sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeHealthcareEntitiesTests.cs

```diff
@@ -133,6 +133,10 @@ public async Task AnalyzeHealthcareEntitiesTest()
                     Assert.AreEqual(5, role.Entity.Length);
                 }
             }
+
+            // Check that the FHIR bundle is not null, but empty.
+            Assert.IsNotNull(result1.FhirBundle);
+            Assert.AreEqual(result1.FhirBundle.Count, 0);
```

Comment:
Nit: with NUnit, the first argument is the expected value (changes how the error message is when the assertion fails) unless you use `Assert.That`.